pylint/checkers/dunder_methods.py

```diff
@@ -193,4 +254,4 @@ def visit_call(self, node: nodes.Call) -> None:
 
 
 def register(linter: PyLinter) -> None:
-    linter.register_checker(DunderCallChecker(linter))
+    linter.register_checker(DunderChecker(linter))
```

Comment:
could also split the two checks into their own checker classes if that makes which checker uses which names more clear